tests/validation/tests/v3_api/test_rke_cluster_provisioning.py

```diff
@@ -716,11 +714,25 @@ def validate_rke_dm_host_2(node_template,
             "worker": True,
             "quantity": 3}
     nodes.append(node)
+    return nodes
+
+
+def validate_rke_dm_host_2(node_template,
+                           rancherKubernetesEngineConfig=rke_config):
+    client = get_user_client()
+    nodes = validate_rke_dm_host_2_build_nodes(node_template)
     cluster, node_pools = create_and_vaildate_cluster(
         client, nodes, rancherKubernetesEngineConfig)
     cluster_cleanup(client, cluster)
 
 
+def validate_rke_dm_host_2_no_cluster_cleanup(node_template,
+                           rancherKubernetesEngineConfig=rke_config):
+    client = get_user_client()
+    nodes = validate_rke_dm_host_2_build_nodes(node_template)
+    create_and_vaildate_cluster(client, nodes, rancherKubernetesEngineConfig)
+
+
```

Comment:
There is no need for validate_rke_dm_host_2_no_cluster_cleanup method. cluster_cleanup method in existing method validate_rke_dm_host_2 that can be modified to not call cluster_cleanup by passing an additional parameter to this method - attemptDelete with default to "True" which can be overridden to false for your workflow.

Reply:
This one is really tricky as the cluster provisioning methods cleanup the cluster based on that variable and if I set it to `False` I will end with a cluster not being tear down at the end of the automation execution having to manually clean up the resources.

So this attempt was to make sure Im not removing the cluster after the original `validate_rke_dm_host_2` because the backup tests happen **after** the initial cluster validations. And cleanup the cluster after the backup tests.